tests/spot/convert/test_convert_trade_history.py

```diff
@@ -0,0 +1,48 @@
+import responses
+from urllib.parse import urlencode
+from tests.util import random_str
+from tests.util import mock_http_response
+from tests.util import current_timestamp
+from binance.spot import Spot as Client
+from binance.error import ParameterRequiredError
+
+mock_item = {"key_1": "value_1", "key_2": "value_2"}
+key = random_str()
+secret = random_str()
+
+
+params = {
+    "startTime": current_timestamp(),
+    "endTime": current_timestamp(),
+}
+
+
+def test_convert_trade_history_without_startTime():
+    """Tests the API endpoint to retreive convert trade history without startTime"""
```

Comment:
typo, should be "retrieve"

Reply:
Yes, a clear type... copy pasted 3 times :)